src/Fantomas.Tests/KeepIfThenInSameLineTests.fs

```diff
@@ -131,3 +131,25 @@ else
     multi
     line
 """
+
+[<Test>]
+let ``MaxIfThenElseShortWidth not exceeded, 825`` () =
+    let c = { config with
+               MaxIfThenElseShortWidth = 100 }
+               
+    formatSourceString false """if foo then
+    bar
+else if fooo then
+    bar
+else if foooo then
```

Comment:
change this one to `elif` please. So both cases are covered.
This may seem like a nitpick now, but completeness is key to detect regressions later.